src/main/com/fulcrologic/rad/report.cljc

```diff
@@ -313,7 +317,7 @@
                                           {:keys [params]} event-data
                                           path                (conj report-ident :ui/parameters)
                                           controls            (report-options env :com.fulcrologic.rad.control/controls)
-                                          initial-sort-params (or (report-options env ::initial-sort-params) {})
+                                          initial-sort-params (initial-sort-params-with-defaults env)
```

Comment:
This was overriding the default value the second time the report was opened. `:initialize` was only being called once.